pruner/find.go

```diff
@@ -31,7 +31,7 @@ func (s *Service) findPruneableHeaders(
 		return nil, err
 	}
 
-	if lastPruned.Height() == estimatedCutoffHeight {
```

Comment:
Could you please add explanation on why estimated height needs to be higher than lastPruned by at least 2? It seems odd, so comment will help

Reply:
Ah yes sure.

Reply:
done https://github.com/celestiaorg/celestia-node/pull/3828/commits/150e9c900b3e6b5d79e5fb5d242207a8df70fc6e